packages/Kokkos/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -92,14 +92,26 @@ KOKKOS_FUNCTION void spatial_query( BVH<NO> const bvh,
         {
             indices[pos] = BVHQuery<NO>::getIndex( bvh, node );
             ++pos;
+#if HAVE_DTK_DBC
+            if ( pos > n_max_indices )
+                printf( "Increase the size of indices array\n" );
```

Comment:
I think this is meant to emulate the DTK macros but these throw exceptions which is not possible on the device.